maths/sin.py

```diff
@@ -0,0 +1,64 @@
+"""
+Calculate sin function.
+
+It's not a perfect function so I am rounding the result to 10 decimal places by default.
+
+Formula: sin(x) = x - x^3/3! + x^5/5! - x^7/7! + ...
+Where: x = angle in randians.
+
+Source:
+    https://www.homeschoolmath.net/teaching/sine_calculator.php
+
+"""
+
+from math import factorial, radians
+
+
+def sin(
+    angle_in_degrees: float, accuracy: int = 18, rounded_values_count: int = 10
+) -> float:
+    """
+    Implement sin function.
+
+    >>> sin(0.0)
+    0.0
+    >>> sin(90.0)
+    1.0
+    >>> sin(180.0)
+    0.0
+    >>> sin(270.0)
+    -1.0
+    >>> sin(0.68)
+    0.0118679603
```

Comment:
but this seems to be incorrect? `sin(0.68) = 0.628793`

Reply:
The `math.sin` function in python accepts an angle measured in radians, but my sin function accepts an angle measured in degrees.

To get the right result use:
```python
>>> import math
>>> math.sin(math.radians(0.68))
0.011867960298537237
```